bumperbot_detection/LitterPoint litter_point
bool duplicate